test/integration/features/story_health.feature

```diff
@@ -26,9 +26,9 @@ Feature:
     Scenario: Start CRC on Windows
         Given executing "crc setup" succeeds
         When starting CRC with default bundle and nameserver "10.75.5.25" succeeds
-        Then stdout should contain "CodeReady Containers instance is running"
+        Then stdout should contain "Started the OpenShift cluster"
         And executing "crc oc-env | Invoke-Expression" succeeds
-        When with up to "4" retries with wait period of "2m" command "crc status" output should contain "Running (v4."
+        When with up to "4" retries with wait period of "2m" command "crc status" output matches ".*Running \(v\d+\.\d+\.\d+.*\).*"
```

Comment:
@jsliacan Does it updated for other platform also with regex?

Reply:
@praveenkumar other platforms already have regex, this was just missed when #543 went in after that change.